beavy/jsbeavy/main.jsx

```diff
@@ -38,5 +39,5 @@ if (window.PRELOAD.CURRENT_USER) initData.CURRENT_USER = window.PRELOAD.CURRENT_
 const store  = configureStore(initData);
 
 
-React.render(<Root application={Application}
+ReactDom.render(<Root application={Application}
```

Comment:
React with its release 0.14 has officially been split into react + different renderers (for the DOM, for native, the canvas and so on). See the release notes to learn more: https://facebook.github.io/react/blog/2015/10/07/react-v0.14.html